File: ELEC881_Copilot/src/Function Name Masked/Java script (JS)/q18-count-numbers-with-unique-digits.js
var abc = function(n) {
    if (n == 0) return 1;
    if (n == 1) return 10;
    var res = 10;
    var count = 9;
    for (var i = 2; i <= n; i++) {
        count *= 11 - i;
        res += count;
    }
    return res;
}

File: ELEC881_Copilot/src/Function Name Masked/Python/q18-count-numbers-with-unique-digits.py
class abc(object):
    def cbc(self, n):
        if n == 0:
            return 1
        if n == 1:
            return 10
        if n > 10:
            return self.cbc(10)
        res = 10
        for i in range(1, n):
            res *= 10 - i
        return res + self.cbc(n - 1)

